docs/architecture/microservices/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md

```diff
@@ -401,7 +401,7 @@ Environment variables are created and initialized in different ways, depending o
 
 The following example shows an .env file like the [.env](https://github.com/dotnet-architecture/eShopOnContainers/blob/master/.env) file for the eShopOnContainers application.
 
-```
+```console
```

Comment:
this is not really console. Maybe change it to env? I believe docs will render anything we put there but it won't colorize for unknown identifiers 

```suggestion
```env
```

Reply:
Good idea! That seems to work and is more accurate.